### PR TITLE
docker-machine-driver-xhyve: Update 0.3.0 and without-qcow2

### PR DESCRIPTION
**Pull request by zchee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/docker-machine-driver-xhyve.rb [hunk 1]
```diff
@@ -14,29 +14,45 @@ class DockerMachineDriverXhyve < Formula
     sha256 "dd8efbfed1d526d159355ed06273540c4e4db0ce3e683e475b7985efaf3e5084" => :yosemite
   end
 
+  option "without-qcow2", "Do not support qcow2 disk image format"
+
   depends_on :macos => :yosemite
   depends_on "go" => :build
   depends_on "docker-machine" => :recommended
+  depends_on "opam" if build.with? "qcow2"
```
Comment (MikeMcQuaid) [inline Formula/docker-machine-driver-xhyve.rb]: Could put these both in a single `if build.with? "qcow2"`

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/docker-machine-driver-xhyve.rb (hunk 1 above)
Comment (zchee) [inline Formula/docker-machine-driver-xhyve.rb]: It means below?
```ruby
if build.with? "qcow2"
  depends_on "opam"
  depends_on "libev"
end
```

Comment (MikeMcQuaid) [inline Formula/docker-machine-driver-xhyve.rb]: Yep 👍

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**